ERROR - 2025-01-08 00:56:28 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'hope', 3306, '', 0)
#1 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\hope\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\hope\app\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\hope\app\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar(Array)
#6 C:\xampp\htdocs\hope\system\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
#7 C:\xampp\htdocs\hope\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\xampp\htdocs\hope\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\hope\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\hope\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\hope\app\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\hope\app\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar(Array)
#5 C:\xampp\htdocs\hope\system\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
#6 C:\xampp\htdocs\hope\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#7 C:\xampp\htdocs\hope\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\hope\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-08 00:56:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `contrasena`, `edad`) VALUES (:nombre:, :correo:, :contrasena:, :edad:)', [...], false)
 3 APPPATH\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert([...])
 4 APPPATH\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-08 02:36:14 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-01-08 02:36:14 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\HOPE\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp\htdocs\HOPE\system\Helpers\number_helper.php(56): format_number(8, 2, NULL, Array)
#1 C:\xampp\htdocs\HOPE\system\Debug\BaseExceptionHandler.php(152): number_to_size(8, 2)
#2 C:\xampp\htdocs\HOPE\app\Views\errors\html\error_exception.php(384): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(8388608)
#3 C:\xampp\htdocs\HOPE\system\Debug\BaseExceptionHandler.php(248): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\HOPE\system\Debug\BaseExceptionHandler.php(251): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\HOPE\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\HOPE\system\Debug\Exceptions.php(157): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\HOPE\system\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\HOPE\system\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\HOPE\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2025-01-08 01:48:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '1006', 'hope', 3306, '', 0)
#1 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\hope\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\hope\app\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\hope\app\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar(Array)
#6 C:\xampp\htdocs\hope\system\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
#7 C:\xampp\htdocs\hope\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#8 C:\xampp\htdocs\hope\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\hope\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\hope\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\hope\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\hope\system\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\hope\app\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\hope\app\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar(Array)
#5 C:\xampp\htdocs\hope\system\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
#6 C:\xampp\htdocs\hope\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#7 C:\xampp\htdocs\hope\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\hope\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2025-01-08 01:48:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2302): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre_usuario`, `correo`, `contrasena`, `edad`) VALUES (:nombre_usuario:, :correo:, :contrasena:, :edad:)', [...], false)
 3 APPPATH\Models\Usuarios_model.php(19): CodeIgniter\Database\BaseBuilder->insert([...])
 4 APPPATH\Controllers\Usuarios.php(35): App\Models\Usuarios_model->registrar([...])
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Usuarios->registrar()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
